frontends/ol-util/src/components/PageBanner.tsx

```diff
@@ -76,7 +76,6 @@ const BannerPageWrapper = styled.div`
 `
 
 const BannerPageHeader = styled.div`
-  margin-bottom: 20px;
   ${imageWrapperHeight};
 `
```

Comment:
See https://github.com/mitodl/open-discussions/pull/3619/commits/241ed7c555378d9d0a602c6211c9959ee0d6798b 

> This change should have no visible effect on open-discussions and simplies the css there, and makes BannerPageHeader easier to reuse.
> 
> In open-discussions, BannerPageHeader is used in 6 places.
>
> - in 5 of the 6 places, the margin-bottom: 20px was doing nothing because the sibling element had class `.main-content`, which specifies margin-top: 20px. And the adjacent 20px margins were being collapsed.
>
>    For info about collapsing margins, see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing or or https://www.joshwcomeau.com/css/rules-of-margin-collapse/ for a deep dive.
> - In ChannelHeader, the 20px was doing something, but only because `.main-content.channel-page` specified 0px margin. So just remove that override, and now 6/6 of the bottom-margins were unncessary.